### PR TITLE
Animation Cleanup: En_T*

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/libultra/vimodes/vimodentschpf1.c [hunk 1]
```diff
@@ -21,15 +21,15 @@ OSViMode osViModeNtscHpf1 = {
       {
           // [0]
           ORIGIN(1280),       // origin
-          SCALE(0.5, 0.5),    // 0x2000800, // yScale
+          SCALE(0.5f, 0.5f),  // 0x2000800, // yScale
```
[the remark this answers is not in the document]
Comment (AngheloAlf) [reply src/libultra/vimodes/vimodentschpf1.c:24]: It is probably fine

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/actors/ovl_En_Talk_Gibud/z_en_talk_gibud.c [hunk 2]
```diff
@@ -114,21 +97,39 @@ ActorInit En_Talk_Gibud_InitVars = {
...
+    { &gGibdoRedeadGrabEndAnim, 0.5f, 0.0f, 0.0f, ANIMMODE_ONCE_INTERP, 0.0f }, // EN_TALK_GIBUD_ANIM_GRAB_END
+    { &gGibdoRedeadGrabStartAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },     // EN_TALK_GIBUD_ANIM_GRAB_START
+    { &gGibdoRedeadLookBackAnim, 0.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },      // EN_TALK_GIBUD_ANIM_LOOK_BACK
+    { &gGibdoRedeadWipingTearsAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f }, // EN_TALK_GIBUD_ANIM_CROUCH_WIPING_TEARS
+    { &gGibdoRedeadSobbingAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },     // EN_TALK_GIBUD_ANIM_CROUCH_CRYING
+    { &gGibdoRedeadDeathAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // EN_TALK_GIBUD_ANIM_DEATH
+    { &gGibdoRedeadDamageAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },      // EN_TALK_GIBUD_ANIM_DAMAGE
+    { &gGibdoRedeadStandUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },     // EN_TALK_GIBUD_ANIM_CROUCH_END
+    { &gGibdoRedeadIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },        // EN_TALK_GIBUD_ANIM_IDLE
+    { &gGibdoRedeadWalkAnim, 0.4f, 0.0f, 0.0f, ANIMMODE_LOOP_INTERP, -8.0f }, // EN_TALK_GIBUD_ANIM_WALK
```
Comment (isghj5) [inline src/overlays/actors/ovl_En_Talk_Gibud/z_en_talk_gibud.c:129]:
```suggestion
    { &gGibdoRedeadWipingTearsAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },   // EN_TALK_GIBUD_ANIM_CROUCH_WIPING_TEARS
    { &gGibdoRedeadSobbingAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },       // EN_TALK_GIBUD_ANIM_CROUCH_CRYING
    { &gGibdoRedeadDeathAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },         // EN_TALK_GIBUD_ANIM_DEATH
    { &gGibdoRedeadDamageAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },        // EN_TALK_GIBUD_ANIM_DAMAGE
    { &gGibdoRedeadStandUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // EN_TALK_GIBUD_ANIM_CROUCH_END
    { &gGibdoRedeadIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },          // EN_TALK_GIBUD_ANIM_IDLE
    { &gGibdoRedeadWalkAnim, 0.4f, 0.0f, 0.0f, ANIMMODE_LOOP_INTERP, -8.0f },   // EN_TALK_GIBUD_ANIM_WALK
```

Comment (isghj5) [reply src/overlays/actors/ovl_En_Talk_Gibud/z_en_talk_gibud.c:129]: Minor nit, I like enums all on the same line... unless formater says no..

File: src/overlays/actors/ovl_En_Test3/z_en_test3.c [hunk 3]
```diff
@@ -981,12 +1028,12 @@ void func_80A40678(EnTest3* this, PlayState* play) {
     struct_80A41828* sp3C;
     ScheduleOutput scheduleOutput;
 
-    this->unk_D80 = ((this->unk_D88 == 20) || (this->unk_D88 == 10) || (this->unk_D88 == 9)) ? 3
-                    : Play_InCsMode(play)                                                    ? 0
+    this->unk_D80 = ((this->scheduleResult == 20) || (this->scheduleResult == 10) || (this->scheduleResult == 9)) ? 3
+                    : Play_InCsMode(play)                                                                         ? 0
                                           : R_TIME_SPEED + ((void)0, gSaveContext.save.timeSpeedOffset);
```
Comment (isghj5) [inline src/overlays/actors/ovl_En_Test3/z_en_test3.c:1033]: what a mess formater made of this...

**Review by isghj5 — APPROVED** [review]

**Comment by starlightgemini214** [comment]
?ecivda ynA  .detartsurf oS !lla staht tub llits wob ym evah I .niaga revo gnihtyreve od ot evah dna thguac i syraf yarts eht lla tsol i ekil skool ti elpmet eht retne i nehw woN .ssob elpmet llafdoow eht taeb tsomla I

**Comment by hirottau302** [comment]
)"!eeh eeh eeH ?loof elttil ,emit erom deen uoy diD" ekil yas lliw ti ti esu uoy emit yreve dna ,emiT fO gnoS eht deyalp uoy taht swonk ti( secnereffid emit eht htiw smelborp on sah ti ekil uoy swollof yllautca ti taht dnA "!denibmoc elpoep esoht lla naht reggib si thgim yM !looF !eeh eeh eeH ?elpmeT daehwonS detelpmoc uoy oS" ekil ,emag eht tuohguorht uoy gnitnuah ksaM sarojaM fo noitidda eht devol evah dluow I sP ?thguoht tnereffid a sah yllautca rO ?sthguoht emas eht dah ohw ;gniksa mI won dnA.ytpme sti ekil sleef ti ecin os syalp dna llew os skrow emag eht ecnis )?enoemos ,tseuQ retsaM( selzzup fo tnuoma eht esaercni ,oslA .sessob eht gnivorpmi dna ni snoegnud artxe emos gnittup ekiL .retteb hcum os enod evah dluoc odnetniN :si tniop yM ?tniop eht stahw ,gniksa eruoy ,oS .gnitseretni yrev sgniht sekam )semitemos( emit fo erusserp eht dna gnitseretni yrev gniyalp noegnudretni sekam ti dna looc yrev era s'cpn dna segalliv dna saera ehT .dnah rehto eht no emosewarepus erehw tolp dna enil yrots ehT .)traW( ssob laer eht naht nuf erom saw ssobdim eht yllautcA .dekcus ssoB eht dias I sa dna "noitanrat ,kcab teg ot smoor evif hguorht og ot deen I won rodirroc gnorw eht tog I spoO" ekil saw taB taerG tuB .nuf stI .taht ekil sgniht dna selpmeT retaW evol I ,naem I .yalp ot gniyonna yrev saw elpmeT yaB taerG ,oslA?smrow dnas niaga yhw dna eman daB = dlonniwT elttaB gniroB = groyG rennur diputS = thoG soahC = awlodO .gnitioppasid erew sessob eht yllaicepse dnA ?snoegnud 4 ,naem I .yrots dna scinahcem semag eht htiw erom hcum os od dluoc yeht ekil sleef tI .detnioppasid mI yllautca dna ksaM sarojaM deyalp I

**Comment by ginomeelectra202** [comment]
I asked if u can go back in time and keep the golden dust a few days ago and now it's gone and I asked how to get the homebrew channel and the day after I asked it it was gone

**Comment by altoidmerope294** [comment]
ksam aroz eht evah i tub ,meht dnif tnac i dna enog era sgge aroz ehT

**Comment by yemenalcyone230** [comment]
.kcuts yllacissab mi os dna won od ot tahw wonk tnod i .krow tndid ti em dlot )yriaf( laet taht retfA .)noom eht pots stniag eht lla eno eht( .detrats noitamina eht dna redro ot htao deyalp i .rewot eht ni pu tnew dna llew deraperp i kcab tog i suht .emit tsrif ta arojam od ot deliaf i oS

**Comment by altoidmerope294** [comment]
I did everything up to the point where that guy gives me Keaton mask and a message. Where is Mama? She is not in the office or at the milk bar. So instead I go to Ikana, meet Kafei behind the rock and he tells me about Sakon. I stand and wait with him and at a little after "Night of the Final Day" appears. Kafei then just fades away and Sakon is nowhere around. So I had to start the task all over again. (I am not patient enough to do this.) It happened again! No Mama, and Kafei disappears. Am I doing something wrong.

**Comment by yemenalcyone230** [comment]
!ereht elbuort gnivah m'i esuoceb )yob eht ot yaw eht lla llor ot evah oy taht eno eht naem I( emag kees dna edih norog eht rof spit yna wonk uoy od ,ot evah I fi dna ??sksam ym lla edart ot semag kees dna edih eht lla yalp ot evah yllaer uoy od ,si noitseuq ym ,ereh melborp a gnivah m'i ,syug ih

**Comment by hirottau302** [comment]
I know certain items specify that they contain enough "power" to transcend through time. But I mean, some other items are just ridiculous.You're telling me, empty bottles can persist through time?So if anyone could answer me this from a lore-wise perspective, I'd be most grateful.

**Review by hensldm — APPROVED** [review]

**Pull request merged**